### PR TITLE
fixes to ivy.Shape

### PR DESCRIPTION
**Pull request by soma2000-lang**
I had by mistake wrote tests for methods not required just removed them +some other fixes

Still there are failures working on those.Currently #18098 and some others are failing.
In continuation from  #15091

### CODE REVIEW

File: ivy/__init__.py [hunk 1]
```diff
@@ -351,27 +351,6 @@ def __sizeof__(self):
     def __dir__(self):
         return self._shape.__dir__()
 
-    def __pow__(self, power, modulo=None):
```
Comment (vedpatwardhan) [inline ivy/__init__.py:354]: Not sure about this, were these methods present in native shape classes like `torch.Size` and `tf.TensorShape`? I'll request @Aarsh2001's review if these functions need to be implemented. Thanks @soma2000-lang 🙂

Comment (soma2000-lang) [reply ivy/__init__.py:354]: @vedpatwardhan No these were not there as seen in 
image
thats why I removed it.

Comment (soma2000-lang) [reply ivy/__init__.py:354]: image

**Review by Aarsh2001 — APPROVED** [review]
Yeah i don't think we need these then, feel free to merge the PR, Thanks !

**Comment by vedpatwardhan** [comment]
Hey @Aarsh2001 @soma2000-lang, just wanted to check if you think this PR is ready to be merged, we can have a look at the CI after the merge conflicts and get it merged if everything seems good. Thanks 😄

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Aarsh2001** [comment]
Closing this PR for now ! Feel free to reopen once you're ready with the changes, thanks 🙂

**Comment by vedpatwardhan** [comment]
I think some of the changes made in this PR are important to add to the repo, could you please take the lead on getting the final few changes and merging if there's not a lot left? Thanks @Aarsh2001 😄

**Comment by Aarsh2001** [comment]
Hey @soma2000-lang, can you please resolve the merge conflicts so we could get this PR merged ? Thanks

**Comment by RickSanchezStoic** [comment]
Hey @Aarsh2001 review this once so that it's good to merge, have resolved the conflicts.

**Pull request merged**